docarray/index/abstract.py

```diff
@@ -116,6 +127,8 @@ def build(self, *args, **kwargs) -> Any:
             """
             ...
 
+        # TODO support subindex in QueryBuilder
```

Comment:
I guess this will be left for future iterations? That is fine, let's just document it properly :)

Reply:
yes lets open issue if we have todo left